FindMe/Recording/RecordingManager.h

```diff
@@ -0,0 +1,23 @@
+//
+//  RecordingManager.h
+//  FindMe
+//
+//  Created by Nicholas Wayoe on 9/14/20.
+//  Copyright © 2020 Nicholas Wayoe. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+@import AVFoundation;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface RecordingManager : NSObject
+
+@property (nonatomic) BOOL isRecording;
+
+- (void)startRecording:(void(^)(NSError * error, BOOL isStarted))completion;
```

Comment:
don't need recording since you're already in the recorder.
```suggestion
- (void)start:(void(^)(NSError * error, BOOL isStarted))completion;
```